sycl/source/detail/device_filter.cpp

```diff
@@ -75,7 +75,7 @@ device_filter::device_filter(const std::string &FilterString) {
     } catch (...) {
       std::string Message =
           std::string("Invalid device filter: ") + FilterString +
-          "\nPossible backend values are {host,opencl,level_zero,cuda,*}.\n"
+          "\nPossible backend values are {host,opencl,level_zero,cuda,rocm,*}.\n"
```

Comment:
```suggestion
          "\nPossible backend values are"
          "{host,opencl,level_zero,cuda,rocm,*}.\n"
```

Reply:
Updated branch with clang format change.
Let me know if there is anything else you wish to rectify.
Thanks